lib/components/overlays/quick_settings/quick_settings_overlay.dart

```diff
@@ -371,16 +335,12 @@ class QsMain extends StatelessWidget
                     return FutureBuilder(
                       future: Battery().batteryLevel,
                       builder: (context, AsyncSnapshot<int?> data) {
```

Comment:
add type annotation directly to FutureBuilder and remove the type from the lambda argument, also possibly rename data to snapshot